Kiss.Bff/Dockerfile

```
@@ -47,7 +47,10 @@ COPY --from=frontendtest /testresults/*.xml .
 
 FROM dotnetbuild AS dotnetpublish
 WORKDIR "/src/Kiss.Bff"
-RUN dotnet publish "Kiss.Bff.csproj" -c Release -o /app/publish /p:UseAppHost=false
+ARG VERSION=0.0.0
+ARG REVISION=dev
+ENV VERSION=$VERSION REVISION=$REVISION
+RUN dotnet publish "Kiss.Bff.csproj" -c Release -o /app/publish /p:UseAppHost=false /p:Version=$VERSION /p:SourceRevisionId=$REVISION /p:InformationalVersion=$VERSION+$REVISION
```

Volgens mij is dit niet nodig: 
```suggestion
RUN dotnet publish "Kiss.Bff.csproj" -c Release -o /app/publish /p:UseAppHost=false /p:Version=$VERSION /p:SourceRevisionId=$REVISION
```

---

https://github.com/Klantinteractie-Servicesysteem/KISS-frontend/blob/11801fe4aface6e81a1722f5bb91fde01094861c/Kiss.Bff/Intern/Environment/EnvironmentController.cs#L39

Informationversion is nodig die gebruiken we in backend om de versie en commit op te halen